lib/peep.ex

```diff
@@ -164,12 +176,85 @@ defmodule Peep do
     end
   end
 
+  def allow_metric?(%Telemetry.Metrics.Summary{} = metric) do
+    Logger.warning("The summary metric type is unsupported. Dropping #{inspect(metric.name)}")
+    false
+  end
+
+  def allow_metric?(%Telemetry.Metrics.Distribution{reporter_options: opts} = metric) do
+    key = :max_value
+
+    case Keyword.get(opts, key) do
+      nil ->
+        true
+
+      n when is_number(n) ->
+        true
+
+      _ ->
+        Logger.warning(
+          "Distributions must have a numeric value assigned to #{inspect(key)} in reporter_options. Dropping #{inspect(metric.name)}"
+        )
+
+        false
+    end
+  end
```

Comment:
Why is it necessary to validate the `:max_value` option for distribution and nothing else?

Reply:
For distributions, the :max_value option is necessary in order to bound the number of buckets for samples.